_scripts/instruction-widget/templates/install/centos.html

```diff
@@ -1,3 +1,5 @@
+<p>If you would like to help test a new way to install Certbot that automatically renews your certificates and stays up-to-date, please choose "snapd" in the system dropdown above.</p>
```

Comment:
We don't want to show this text to CentOS/RHEL 6 users. To fix this, I'd wrap this text with:
```
{{#packaged}}
{{/packaged}}
```